client/pages/forgot.js

```diff
@@ -0,0 +1,81 @@
+import {
+  Button,
+  FormControl,
+  Flex,
+  Heading,
+  Input,
+  Stack,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+const ResetPassword = () => {
+  const [resetPasswordInput] = useState({
+    email: "",
+  });
+
+  const handleChange = (event) => {
+    resetPasswordInput({
```

Comment:
Here `resetPasswordInput` is a state and it is being used as a function, which would result in runtime issues. 

Change it to 
```jsx
setForgotPasswordInput({ email: event.target.value })
```